bin/pulumilocal

```diff
@@ -124,22 +193,79 @@ def set_localstack_pulumi_config(args: argparse.Namespace):
 def main():
     # Parse arguments from call to pulumi CLI that set the stack name and directory
     parser = argparse.ArgumentParser(add_help=False)
-    parser.add_argument("command", help="pulumi primary command",
+    parser.add_argument("command", help="Pulumi command",
                         type=str)
-    parser.add_argument("-s", "--stack", help="pulumi stack name",
+    parser.add_argument("-s", "--stack", help="The name of the stack to operate on. Defaults to the current stack",
                         required=False,
                         type=str)
-    parser.add_argument("-C", "--cwd", help="run in this directory",
+    parser.add_argument("-C", "--cwd", help="Run pulumi as if it had been started in another directory",
                         required=False,
                         type=str)
+    parser.add_argument("--config-file", help="Use the configuration values in the specified file rather than detecting the file name",
+                        required=False,
+                        type=str)
+    parser.add_argument("--non-interactive", help="Disable interactive mode for all commands",
+                        action="store_true",
+                        required=False)
     args, _ = parser.parse_known_args()
 
+    if NON_INTERACTIVE:
+        args.non_interactive = True
+
+    if not args.cwd:
+        args.cwd = "."
+
+    if os.environ.get("CONFIG_STRATEGY") not in CONFIG_STRATEGIES:
+        print(f"Config strategy: {os.environ.get('CONFIG_STRATEGY')}")
+        print(f'Config strategy is not recognised. Falling back to default ("{DEFAULT_CONFIG_STRATEGY}").')
+
     # If this is a pulumi deployment command, update the stack with LocalStack AWS config
-    if args.command in ["up", "destroy", "preview", "cancel"]:
+    if args.command in PROXIED_CMDS:
+        if not args.stack:
+            try:
+                args.stack = subprocess.run(
+                    [PULUMI_CMD, "stack", "--show-name", "--non-interactive", "-C", args.cwd],
+                    env=os.environ,
```

Comment:
I _think_ this is the default

Reply:
Clarify this a bit please, I'm not sure I get what you meant.
- without `--show-name` the `stack` sub-command only displays the help for itself
- `--non-interactive` is necessary because if there is no stack selected pulumi waits for input
- `-C` is to ensure we're in the right folder

Reply:
I mean specifically the line I am commenting on: the 
```python
env=os.environ
```

is the default behaviour.

Reply:
Ah, you are right. I have overlooked this in the docs.

>...these are used instead of the default behavior of inheriting the current process’ environment. ....

Apparently I'm not reading either. 😅